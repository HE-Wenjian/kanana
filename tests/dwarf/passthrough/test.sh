#!/bin/sh

check()
{
  if [ $? -ne 0 ]; then
    exit 1
  fi
}

../../../katana recompile.ksh
check
./test_mod > log
check
diff log test.out
check

