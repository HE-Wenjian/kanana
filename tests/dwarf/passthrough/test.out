Caught int 1
foo caught int 2
